models/user.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function(sequelize, DataTypes) {
+  var User = sequelize.define("User", {
+    email: DataTypes.STRING,
+    password: DataTypes.STRING,
```

Comment:
Change 'password' to 'password_hash'. We haven't gone over this in class yet, but I can explain this to you on Wednesday.